.vsts-ci.yml

```diff
@@ -276,3 +276,35 @@ jobs:
       enablePublishBuildArtifacts: true
       pool:
         vmImage: vs2017-win2016
+        
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - job: Create_GraphViz
+    displayName: Create GraphViz
+    dependsOn:
+      - Windows_NT
+      - Linux
+      - Darwin
+      - Asset_Registry_Publish
+    pool:
+      name: Hosted VS2017
+    condition: succeeded()
+    variables:
+      # Publish-Build-Assets provides: MaestroAccessToken, BotAccount-dotnet-maestro-bot-PAT
+      # DotNet-AllOrgs-Pats provides: dn-bot-all-orgs-pat
+      - group: Publish-Build-Assets
+      - group: DotNet-AllOrgs-Pats
```

Comment:
This is a confusing variable group name and doesn't follow the guidance we have https://github.com/dotnet/core-eng/blob/master/Documentation/Secret-Management/secret-naming-conventions.md#vsts-resources.  I'd be more explicit about what the orgs are (devdiv-dnceng)